### PR TITLE
BIP-613: Fraxtal and Mode Wireup

### PR DESCRIPTION
**Pull request by Tritium-VLK**

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Need to use proper action id's in the A-D paylaods.  They were copied from base. Will sort that out tomorrow and generate new reports.

File: BIPs/2024-W23/BIP-613/BIP-613D.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        "name": "grantRoles",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "roles": "[0x2960d085c4c968d7cad55c0da3f97014525b948fdce990ecaef4e832b5f0b151,0x3f44776af02a9227991da37715b44e45db40735e216b3ab33144859bb6737166]",
+        "account": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f"
+      }
+    },
+    {
+      "to": "0x5DbAd78818D4c8958EfF2d5b95b28385A22113Cd",
```
Comment (Tritium-VLK) [inline BIPs/2024-W23/BIP-613/BIP-613D.json:32]: Need add gauge type for fraxtal

**Comment by gosuto-inzasheru** [comment]
i moved the fraxtal root gauge payload to its own pr: https://github.com/BalancerMaxis/multisig-ops/pull/1003

File: BIPs/2024-W23/BIP-613/inputs.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      "20230409-gearbox-linear-pool-v2",
+      "yearn-linear-pool-v2",
+      "20230410-aave-linear-pool-v5",
+      "20230411-managed-pool-v2"
+    ],
+    "function_caller_map": {
+      "setSwapFeePercentage(uint256)": ["multisigs/lm"],
+      "startAmplificationParameterUpdate(uint256,uint256)": ["multisigs/lm"],
+      "stopAmplificationParameterUpdate()": ["multisigs/lm"],
+      "/pause()": ["multisigs/emergency"],
```
Comment (gosuto-inzasheru) [inline BIPs/2024-W23/BIP-613/inputs.json:21]: is this slash here correct? it seems to interfere with at least the authorizer lookup

File: BIPs/2024-W23/BIP-613/inputs.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      "fraxtal": 252
+    },
+    "deployments": [
+      "weighted-pool-v4",
+      "composable-stable-pool-v6",
+      "composable-stable-pool-v5",
+      "20230409-erc4626-linear-pool-v4",
+      "20230409-gearbox-linear-pool-v2",
+      "yearn-linear-pool-v2",
+      "20230410-aave-linear-pool-v5",
```
Comment (gosuto-inzasheru) [inline BIPs/2024-W23/BIP-613/inputs.json:14]: these five deployments dont exist on neither mode nor fraxtal

**Review by gosuto-inzasheru — COMMENTED** [review]
in comparison with the `inputs.json` from base and avax deployments:
- added `composable-stable-pool-v6`
- updated `batch-relayer-v5` to v6

File: BIPs/2024-W23/BIP-613/inputs.json (hunk 3 above)
Comment (Tritium-VLK) [reply BIPs/2024-W23/BIP-613/inputs.json:14]: The perm granter basically checks and adds whatever is there.  For that reason we just list all the pools that we have ever deployed/seen here and anything that exists gets permissions.  If you look at the end payload, none of them got permissions in the end/were ignored.  You can also look at the action run outputs for more details.

File: BIPs/2024-W23/BIP-613/inputs.json (hunk 2 above)
Comment (Tritium-VLK) [reply BIPs/2024-W23/BIP-613/inputs.json:21]: yeah this uses search_unique() in bal_addresses.  There's also unpause which makes pause not unique but /pause unique.

**Review by gosuto-inzasheru — APPROVED** [review]

**Comment by gosuto-inzasheru** [comment]
seems to match with the permissions we currently have according to https://docs.balancer.fi/reference/authorizer/, except for `disableRecoveryMode`, which i cannot find in the auto generated report here (nor in the auto generated reports for avax nor base, bip 318 and bip 388)

**Comment by Tritium-VLK** [comment]
I think this was done in a PR later.  It kind of doesn't become a thing until recovery mode is set anyway.  See https://github.com/BalancerMaxis/multisig-ops/issues/1009.

**Pull request merged**